vector/src/main/java/im/vector/app/features/location/option/LocationSharingOption.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.location.option
+
+enum class LocationSharingOption {
+    USER_CURRENT, // current user's location
+    USER_LIVE,    // user's location during a certain amount of time
+    PINNED        // static location pinned by the user
```

Comment:
On the form I prefer to have comment above the value, using KDoc for better integration with IDE. So like
```kotlin
/**
 * static location pinned by the user
 */
PINNED
```

Reply:
Okay I will update it.